kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md

```diff
@@ -384,3 +384,40 @@ Sometimes in production, but most likely in testing environments, you might want
 $ kubectl create -f delete-domain-job.yaml
 
 ```
+## Troubleshooting
+
+1. Message: "status on iteration 20 of 20
+pod domain1-create-weblogic-sample-domain-job-4qwt2 status is Pending
+The create domain job is not showing status completed after waiting 300 seconds."  
+* The most likely cause is related to the value of persistentVolumeClaimName, defined in domain-home-on-pv/create-domain-inputs.yaml.  
+To find out if this is the problem, do the following:
+    * execute `kubectl get all --all-namespaces` to find the name of the create weblogic sample domain job.
+    * execute  `kubectl describe pod <name-of-create-weblogic-sample-domain-job>` to see if there is an Event that has text similar to `persistentvolumeclaim "domain1-weblogic-sample-pvc" not found`
+    * find the name of the pvc that was created by executing [create-pv-pvc.sh](../../create-weblogic-domain-pv-pvc/README.md), using `kubectl describe pvc`. It is likely to be `weblogic-sample-pvc`
+
+Change the value of persistentVolumeClaimName to match the name created when you executed [create-pv-pvc.sh](../../create-weblogic-domain-pv-pvc/README.md). Rerun the `create-domain.sh` script with the same arguments as you did before.
+* Verify that the operator is deployed. Use the command
+```
+kubectl  get all --all-namespaces
+```
+and look for lines similar to
+```
+weblogic-operator1   pod/weblogic-operator-
+```
+If you do not find something similar in the output, the WebLogic Operator for Kubernetes may not have been installed completely. Review the operator [installation instructions](../../../../../site/install.md).
+
+2. Message: "ERROR: Unable to create folder /shared/domains"  
+The most common cause is a poor choice of value for weblogicDomainStoragePath in the input file used when you executed 
+```
+create-pv-pvc.sh
+```
+Then you should [delete the resources for your sample domain](../../delete-domain/README.md), correct the value in that file and rerun the commands to create the pv/pvc and the credential before you attempt to rerun
+```
+create-domain.sh
+```
+A correct values for weblogicDomainStoragePath will meet the following requirements
+* owned by the user that started the operator
```

Comment:
Shouldn't this be 'should be owned by the user that's used by the create-domain scripts kubernetes job and weblogic server kubernetes pod images, or should be world accessible (0777)'.